packages/bpk-component-horizontal-nav/src/__snapshots__/BpkHorizontalNav-test.js.snap

```diff
@@ -176,6 +182,7 @@ exports[`BpkHorizontalNav should render correctly with custom "trailingScrollInd
       class="bpk-mobile-scroll-container__scroller"
     >
       <nav
+        aria-label=""
```

Comment:
In reference to the other comment, there should be a tests with both it adding and not adding the aria-label tag